app/index.js

```diff
@@ -1,214 +1,233 @@
-
-		// Copy public folder modules
-		this.directory('public/modules/users');
-
-		// Copy core module files
-		this.directory('public/modules/core/config');
-		this.directory('public/modules/core/controllers');
-		this.directory('public/modules/core/css');
-		this.directory('public/modules/core/img');
-		this.directory('public/modules/core/services');
-		this.directory('public/modules/core/tests');
-		this.copy('public/modules/core/views/home.client.view.html');
-		this.copy('public/modules/core/core.client.module.js');
-
-		// Copy config folder
-		this.mkdir('config');
-		this.mkdir('config/env');
-		this.mkdir('config/sslcerts');
-
-		// Copy config folder content
-		this.directory('config/strategies');
-		this.copy('config/config.js');
-		this.copy('config/init.js');
-		this.copy('config/express.js');
-		this.copy('config/passport.js');
-
-		// Copy project files
-		this.copy('karma.conf.js');
-		this.copy('gruntfile.js');
-		this.copy('server.js');
-		this.copy('Procfile');
-		this.copy('fig.yml');
-		this.copy('Dockerfile');
-		this.copy('generate-ssl-certs.sh');
-		this.copy('README.md');
-		this.copy('LICENSE.md');
-
-		// Copy project hidden files
-		this.copy('bowerrc', '.bowerrc');
-		this.copy('csslintrc', '.csslintrc');
-		this.copy('editorconfig', '.editorconfig');
-		this.copy('jshintrc', '.jshintrc');
-		this.copy('gitignore', '.gitignore');
-		this.copy('slugignore', '.slugignore');
-		this.copy('travis.yml', '.travis.yml');
-	},
-
-	renderArticleExample: function() {
-		// Copy example files if desired
-		if (this.addArticleExample) {
-			// Copy Express files
-			this.copy('app/controllers/articles.server.controller.js');
-			this.copy('app/models/article.server.model.js');
-			this.copy('app/routes/articles.server.routes.js');
-			this.copy('app/tests/article.server.model.test.js');
-			this.copy('app/tests/article.server.routes.test.js');
-
-			// Copy AngularJS files
-			this.directory('public/modules/articles');
-		}
-	},
-
-	renderApplicationEnvironmentConfigFiles: function() {
-		this.template('config/env/_all.js', 'config/env/all.js');
-		this.template('config/env/_development.js', 'config/env/development.js');
-		this.template('config/env/_production.js', 'config/env/production.js');
-		this.template('config/env/_test.js', 'config/env/test.js');
-		this.template('config/env/_secure.js', 'config/env/secure.js');
-	},
-
-	renderAngularApplicationConfigFile: function() {
-		this.template('public/_config.js', 'public/config.js');
-	},
-
-	renderCoreModuleFiles: function() {
-		this.template('public/modules/core/views/_header.client.view.html', 'public/modules/core/views/header.client.view.html');
-	},
-
-	renderApplicationDependenciesFiles: function() {
-		this.template('_package.json', 'package.json');
-		this.template('_bower.json', 'bower.json');
-	}
+  
+  init: function () {
+    this.pkg = yeoman.file.readJSON(path.join(__dirname, '../package.json'));
+
+    this.on('end', function () {
+      clog('');
```

Comment:
I got lazy writing console.log(chalk.color('The output goes here')) across multiple projects, so it's just an npm module that shortens it.